### ISSUE TITLE
absence of P waves

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6167]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://pubmed.ncbi.nlm.nih.gov/16188595/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
